Sources/Momento/CacheClient.swift

```diff
@@ -317,45 +325,45 @@ public class CacheClient: CacheClientProtocol {
     /**
      Deletes a cache and all items in it.
      - Parameter cacheName: name of the cache to be deleted
-     - Returns: CacheDeleteResponse representing the result of the delete cache operation.
+     - Returns: DeleteCacheResponse representing the result of the delete cache operation.
      Pattern matching can be used to operate on the appropriate subtype.
     ```
      switch response {
-     case let responseError as CacheDeleteError:
+     case let responseError as DeleteCacheError:
```

Comment:
are the code examples in the docstrings going to have to change into enum case matching?

```
switch response {
case DeleteCacheError.error(let e):
   ...
case DeleteCacheError.success(let s):
   ...
}
```

Reply:
I can pick up the task of updating the docstrings here while doing the other docstrings work though